modules/apps/frontend-data-set/frontend-data-set-views-web/src/main/java/com/liferay/frontend/data/set/views/web/internal/portlet/FDSViewsPortlet.java

```diff
@@ -689,33 +688,31 @@ private ObjectDefinition _createFDSViewObjectDefinition(
 			fdsViewObjectDefinition.getObjectDefinitionId(),
 			labelObjectField.getObjectFieldId());
 
-		if (FeatureFlagManagerUtil.isEnabled("LPD-10735")) {
-			ObjectField defaultVisualizationModeObjectField =
-				ObjectFieldUtil.createObjectField(
-					ObjectFieldConstants.BUSINESS_TYPE_TEXT,
-					ObjectFieldConstants.DB_TYPE_STRING, true, false, null,
-					_language.get(locale, "default-visualization-mode"),
-					"defaultVisualizationMode", false);
-
-			_objectFieldLocalService.addCustomObjectField(
-				defaultVisualizationModeObjectField.getExternalReferenceCode(),
-				userId,
-				defaultVisualizationModeObjectField.getListTypeDefinitionId(),
-				fdsViewObjectDefinition.getObjectDefinitionId(),
-				defaultVisualizationModeObjectField.getBusinessType(),
-				defaultVisualizationModeObjectField.getDBType(),
-				defaultVisualizationModeObjectField.isIndexed(),
-				defaultVisualizationModeObjectField.isIndexedAsKeyword(),
-				defaultVisualizationModeObjectField.getIndexedLanguageId(),
-				defaultVisualizationModeObjectField.getLabelMap(), false,
-				defaultVisualizationModeObjectField.getName(),
-				defaultVisualizationModeObjectField.getReadOnly(),
-				defaultVisualizationModeObjectField.
-					getReadOnlyConditionExpression(),
-				defaultVisualizationModeObjectField.isRequired(),
-				defaultVisualizationModeObjectField.isState(),
-				defaultVisualizationModeObjectField.getObjectFieldSettings());
-		}
+		ObjectField defaultVisualizationModeObjectField =
+			ObjectFieldUtil.createObjectField(
```

Comment:
@chestofwonders Here we need to move lines 692-696 into this [ArrayList](https://github.com/liferay-frontend/liferay-portal/pull/4113/files#diff-39d94b45a2c32f4cf77847688762373395176294708092a3c04cf6f73bfa59f6R634) and delete everything else.

Reply:
Done!